integration-tests/aqua/lib.aqua

```diff
@@ -68,6 +70,12 @@ func get_counter(spell_id: string) -> U32Value:
         count <- Spell.get_u32("counter")
     <- count
 
+func get_location(spell_id: string) -> LocationValue:
```

Comment:
I personally don't like such wrappers. I find it tiresome to write them all the time I want to test some service's API. I think we'd better have an option that would allow us to run `on HOST_PEER_ID` from cli.